packages/teams-js/src/private/privateAPIs.ts

```diff
@@ -119,6 +119,24 @@ export function registerUserSettingsChangeHandler(
   registerHandler('userSettingsChange', handler, true, [settingTypes]);
 }
 
+/**
+ * @hidden
+ * register a handler to be called when a user's 3P cloud storage provider details changes. The changed setting type & value is provided in the callback.
+ *
+ * @param settingTypes - List of user setting changes to subscribe
```

Comment:
should be "scenarioTypes", and should probably @link to the CloudStorageProvider documentation

Reply:
That's correct. Updated the same.